rero_ils/modules/documents/jsonschemas/documents/document_temporal_coverage-v0.0.1.json

```diff
@@ -40,13 +40,24 @@
               "title": "Date",
               "description": "Enter a date in the format (+/-)yyyy. You can add the month, day or time as follows: (+/-)yyyy-mm-ddTHH-MM-SS.",
               "type": "string",
-              "pattern": "^(\\+|\\-)\\d+(-\\d{2}-\\d{2}T[0-2][0-9]-[0-6][0-9]-[0-2][0-9])?$",
+              "pattern": "^(\\+|\\-)\\d+(-\\d{2}-\\d{2})?(T[0-2][0-9]-[0-6][0-9]-[0-2][0-9])?$",
               "form": {
                 "placeholder": "Example: +1945-05-08"
               }
             },
             "period_code": {
-              "$ref": "#/definitions/period_code"
+              "title": "Period codes",
+              "minItems": 0,
+              "type": "array",
+              "items": {
+                "title": "Period code",
+                "description": "Enter a 2 character code from the MARC time period table.",
+                "type": "string",
+                "pattern": "^([a-z][0-9-])$",
```

Comment:
I think it is better to force the 4 characters, as it is a MARC standard: *"A single year or a period within a decade is coded by repeating the appropriate two-character code to create the required four characters."*